### ISSUE TITLE
Why not batch normalization?

### ISSUE THREAD

**Issue by huangbiubiu** [wy1iu/sphereface#102]
I found that SphereFace network removed batch normalization layer? Why?

**Comment by jimeffry** [comment]
Maybe the batch_norm layer  could give a fixed value , when the result of the layer is Nan.
